### PR TITLE
Asserions makeInput

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

**Comment by grewered** [comment]
for lines 67, 102: you can check for missings with checkmate as well (and get TRUE, FALSE output: with test_data_frame(x, any.missing = FALSE)

**Comment by grewered** [comment]
see #25 (data frames, type character)

File: R/makeInput.R [hunk 1]
```diff
@@ -1,5 +1,7 @@
 
 makeInputLists <- function (values, subunits, units, recodedData = TRUE) {
+  lapply(c(values, subunits, units), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/makeInput.R:3]: please use list(values, subunits, units)

File: R/makeInput.R [hunk 2]
```diff
@@ -20,6 +22,7 @@ makeInputLists <- function (values, subunits, units, recodedData = TRUE) {
 ## doppelte Eintraege fliegen raus
 
 makeInputCheckData <- function (values, subunits, units) {
+  lapply(c(values, subunits, units), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/makeInput.R:25]: please use list(values, subunits, units)

File: R/makeInput.R [hunk 3]
```diff
@@ -40,6 +43,7 @@ makeInputCheckData <- function (values, subunits, units) {
 ## fuer recodeData benoetigte Inputs erstellen
 
 makeInputRecodeData <- function (values, subunits) {
+  lapply(c(values, subunits), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/makeInput.R:46]: please use list(values, subunits)

**Review by sachseka — CHANGES_REQUESTED** [review]

**Review by sachseka — APPROVED** [review]

**Pull request merged**